assignments/assignment01.rb

```diff
@@ -38,12 +38,21 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
-  words = s.split
-  caps = []
-  words.each do |word|
-    caps << word.capitalize
+
+  # first downcase the entire string
+  s.downcase!
```

Comment:
use of the destructive variation of downcase and capitalize is discouraged and not required for this problem. I encourage you to seek solutions that don't use the ! methods.